docs/howto/configure.md

```diff
@@ -108,3 +108,7 @@ Additional kwargs to `compile` functions take higher precedence. So if you set t
   * Enable promotions in encrypted shifts instead of casting in runtime. See [Bitwise#Shifts](../tutorial/bitwise.md#Shifts) to learn more.
 * **composable**: bool = False,
   * Specify that the function must be composable with itself.
+* **relu_on_bits_threshold**: int = 7,
+  * Bit-width to start implementing the ReLU extension with [fhe.bits](../tutorial/bit_extraction.md).
+* **relu_on_bits_chunk_size**: int = 2,
```

Comment:
maybe it could be renamed as
relu_on_bits_max_tlu_precision
since it's not clear if the chunks oincludes the sign or not

Reply:
I think chunk_size is better as it clearly show how the input will be split.